ttnn/cpp/ttnn/operations/matmul/device/matmul_op.hpp

```diff
@@ -186,6 +188,29 @@ Matmul create_matmul_struct(
     const struct Matmul &parameters
 );
 
+operation::ProgramWithCallbacks matmul_multi_core_reuse_mcast_1d_optimized_helper(
```

Comment:
These helpers had to be moved down here, so that `MatmulProgramConfig` can be used to minimize the number of parameters in helper functions.